demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -62,3 +64,36 @@ def is_no_rolename(self):  # type: () -> bool
             self.is_valid = False
             return False
         return True
+
+    def is_condition_branches_handled_correctly(self):  # type: () -> bool
+        """Check whether the playbook conditional tasks has all optional branches handled
+
+        Return:
+            bool. if the Playbook handles all condition branches correctly.
+        """
+        tasks = self.current_file.get('tasks', {})
+        if tasks:
+            for task in tasks.values():
+                if task.get('type') == 'condition':
+                    # default condition should always exist
+                    task_condition_labels = {'#default#'}
+                    for condition in task.get('conditions', []):
+                        label = condition.get('label')
+                        if label:
+                            task_condition_labels.add(label)
+                    next_tasks = task.get('nexttasks', {})
+                    for next_task_branch, next_task_ids in next_tasks.items():
+                        try:
+                            if next_task_ids:
+                                task_condition_labels.remove(next_task_branch)
+                        except KeyError as e:
+                            print_error('Playbook has conditional task with unreachable next task: {}'.format(str(e)))
```

Comment:
use f-strings
what will the error message look like? will it be any indicative?

Reply:
Yeah, the str(e) of a key error is the key itself, so it will print the task ID.

Reply:
please post screenshot of example

Reply:
see comment below